### PR TITLE
Morse code

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
- Created a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase
- Created a method to decode an entire word in Morse code, takes a string parameter, and returns the string representation. Every character in a word will be separated by a single space.
- Created a method to decode the entire message in Morse code, take a string parameter, and return the string representation. Every word will be separated by 3 spaces

### CODE REVIEW

File: morse.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  str
+end
+
+def decode_msg(msg)
+  message = ''
+  msg.strip.tr('/', ' ').split.each do |element|
+    message += decode_word(element)
+  end
+  message
+end
```
Comment (hiromataba) [inline morse.rb]: - [ ] Your Code is working but there is one more thing you need to fix, Your message is not adding any space to separate the words. you were supposed to return `A BOX FULL OF RUBIES` but look at your output :point_down: 

image

Kindly fix that :pray:

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave:  Danieldotcomcoder , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: morse.rb (hunk 1 above)
Comment (Danieldotcomcoder) [reply morse.rb]: Thank you , I did fix that 👍

**Review by Lameck1 — APPROVED** [review]
Hi, @Danieldotcomcoder & @karelvanoordt,

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**